packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -272,21 +280,6 @@ export default class TSGenerator extends CodeGeneratorLanguage {
     });
 
     // Add our core API methods for controlling auth, servers, and various configurable abilities.
-    sdkSource.addInterface({
```

Comment:
This interface exists in `api/core` so I can just load it for the SDK from that instead of having to duplicate it.